workers/__tests__/uploader.spec.jsx

```diff
@@ -136,12 +136,14 @@ describe('Uploader', () => {
       expect(mockWorker.postMessage).toHaveBeenNthCalledWith(1, {
         type: 'UPLOAD_STATUS',
         upload: {
-          status: 'active',
-          url: 'http://localhost',
-          project_name: 'labors',
-          file: {}, // Can't expect a File object back from the server
-          file_size: file.size,
-          file_name: file.name
+          upload: {
```

Comment:
Is this double nesting the upload object?  Seems like your change below makes this unnecessary.

Reply:
Yeah, this double nests `upload`. I think it's actually required, because without it downstream actions after selecting a file have `undefined` `upload` objects, and Metis rejects the blobs.

Reply:
got it, kk.